static-assets/components/cstudio-admin/mods/graphiql.js

```diff
@@ -32,8 +31,8 @@ var wfStates = [];
 YAHOO.extend(CStudioAdminConsole.Tool.GraphiQL, CStudioAdminConsole.Tool, {
 	renderWorkarea: function() {
 		var workareaEl = document.getElementById("cstudio-admin-console-workarea");
-		
-		workareaEl.innerHTML = 
+
+		workareaEl.innerHTML =
```

Comment:
`$('#cstudio-admin-console-workarea').html('<div id="graphContainer"/>');`

Notice the single quotes and camelcase id

Reply:
Please do as shown in the comment